core/src/block.rs

```diff
@@ -405,10 +405,10 @@ pub struct ValidSignedBlock {
     pub event_recommendations: Vec<Event>,
 }
 
-impl ValidSignedBlock {
+impl SignedBlock {
     /// Commit block to the store.
     #[inline]
-    pub fn commit(self) -> CommittedBlock {
+    pub fn commit_unchecked(self) -> CommittedBlock {
```

Comment:
please add a description in the doc which invariants must hold true if this function is used

Reply:
@mversic there is no invariants really because in some cases we just want to obtain `ComittedBlock` bypassing signatures verification.

Reply:
I would add a comment saying that when calling this function it is users responsibility to make sure there is enough of valid signatures and to prefer using `commit` instead when possible

Reply:
sure